src/helix-ui/styles/components/layouts.less

```diff
@@ -197,115 +197,3 @@ body.hxHorizontal {
     grid-area: footer;
   }
 }//body.hxHorizontal
-
```

Comment:
Moved all `hx-div` related styles to `src/helix-ui/styles/components/box.less` (where `hx-div` is documented)